server/utils/statemanager/drawings_in_progress_state.go

```diff
@@ -34,9 +44,23 @@ func (state drawingsInProgressState) addPrompt(prompts *models.Prompt) error {
 }
 
 func (state drawingsInProgressState) addGameStatusPropertiesForPlayer(player *models.Player, gameStatus *GameStatusResponse) error {
-	gameStatus.CurrentPlayer.AssignedPrompt = &AssignedPrompt{
-		Adjectives: player.AssignedPrompt.Adjectives,
-		Noun:       player.AssignedPrompt.Noun,
+	authorToDrawingMap := map[string]*models.Drawing{}
+	for _, currentDrawing := range state.game.Drawings {
+		authorToDrawingMap[currentDrawing.Author] = currentDrawing
+	}
+	// Mark players who haven't submitted their drawing as having pending actions
```

Comment:
@mick-warehime @apairofcleats what do you think of doing something like this for the voting / prompt states?